lib/tasks/import_prebakedstats.rake

```diff
@@ -20,61 +20,65 @@ namespace :import do
   def import_new_csv_file(habitat, csv_file)
     filename = "#{Rails.root}/lib/data/#{csv_file}"
     CSV.foreach(filename, headers: true, encoding: "utf-8") do |row|
+      name = row['ISO3'] || row['region']
+      geo_entity = fetch_geo_entity(name)
+      next unless geo_entity
+
       if row.select {|k,v| /baseline/i =~ k}.any?
```

Comment:
Could this condition be in a named method? It isn't clear to me what it's doing/why. Excluding all rows without an entry for a baseline column?

Reply:
I've added a comment to explain